src/codecoverage/bot/code_coverage_bot/github.py

```diff
@@ -46,6 +47,13 @@ def mercurial_to_git():
             r = requests.get('{}/gecko-dev/rev/hg/{}'.format(self.hg_git_mapper, mercurial_commit))
 
             if not r.ok:
+                for email in secrets[secrets.REPO_MAPPER_EMAIL_ADDRESSES]:
+                    self.notify_service.email({
+                        'address': email,
+                        'subject': 'Missing commit in the mapper service',
+                        'content': f"Mercurial commit {mercurial_commit} is missing on the mapper service.",
```

Comment:
Here you need to use single quotes, you can see the build failure by clicking on the taskcluster link at the bottom of the PR:
```
./code_coverage_bot/github.py:54:36: Q000 Remove bad quotes
```